src/components/ArticleTemplate.js

```diff
@@ -1,13 +1,16 @@
 import React from "react";
 import { Container, Divider, Header, Placeholder } from "semantic-ui-react";
 
-const ArticleTemplate = (props) => {
+const ArticleTemplate = props => {
   const article = props.article;
 
   return (
     <>
       <Container>
         <div id={`article_${article.id}`}>
+        <Header size="medium" textAlign="left" id="title">
+            {article.city}
```

Comment:
are u showing the article city two times in this template?

Reply:
fixed

Reply:
Very fixed